.github/workflows/main.yml

```diff
@@ -226,3 +226,9 @@ jobs:
   #         artifact_id: ${{ steps.get_artifact_id.outputs.artifact_id }}
   #       env:
   #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Deploying Playground application to ZEIT Now
+        #if: github.ref == 'refs/heads/master'
```

Comment:
Just so that I can test this on this PR, then it will only run on master builds.